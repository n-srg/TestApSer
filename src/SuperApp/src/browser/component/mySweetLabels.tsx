import * as React from 'react';

export class MySweetLabel extends React.Component<myProps, {}> {
	constructor(props) {
		super(props);
	}

	public render() {
		return <div>hi!!{this.props.name}</div>;
	}
}

interface myProps {
	name: string;
}
