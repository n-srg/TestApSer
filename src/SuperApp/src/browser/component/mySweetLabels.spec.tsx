import * as React from 'react';
import * as renderer from 'react-test-renderer';
import { MySweetLabel } from './mySweetLabels';

describe('test my new component', () => {
	it('lets render it and check!', () => {
		const a = renderer.create(<MySweetLabel name="fella!" />).toJSON();

		expect(a).toMatchSnapshot();
	});

	it('paste random screenshot', () => {
		const a = { name: 'good guy!' };

		expect(JSON.stringify(a)).toMatchSnapshot();
	});
});
