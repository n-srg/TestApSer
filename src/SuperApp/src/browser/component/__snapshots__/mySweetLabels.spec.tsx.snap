// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test my new component lets render it and check! 1`] = `
<div>
  hi!!
  fella!
</div>
`;

exports[`test my new component paste random screenshot 1`] = `"{\\"name\\":\\"good guy!\\"}"`;
